src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -205,8 +207,9 @@ private static void PromptRequiredProperties<T>(T manifest, VersionManifest vers
 
                     var currentValue = property.GetValue(manifest);
                     var result = PromptProperty(manifest, currentValue, property.Name);
-                    property.SetValue(manifest, result);
-                    Logger.Trace($"Property [{property.Name}] set to the value [{result}]");
+                    property.SetValue(manifest, result);
+                    string formattedResult = result.ToString().Replace("{", string.Empty).Replace("}", string.Empty);
+                    Logger.Trace($"Property [{property.Name}] set to the value [{formattedResult}]");
                 }
```

Comment:
I don't think this addresses their issues. All the stack traces seem to originate in this call in Program.cs:

    Logger.Error(ex.ToString());

I believe that what's possibly happening is that the exception method has a format string inside of it, for example "{version}". You can repro with:

wingetcreate new [https://foo.com/{version}.exe](https://foo.com/{version}.exe)

Reply:
We should fix this in a separate PR

Reply:
I reverted my changes because your recent PR fixes this issue.

Reply:
(Fixed in separate PR)